x-pack/plugins/ml/server/routes/data_frame_analytics.ts

```diff
@@ -609,12 +609,12 @@ export function dataFrameAnalyticsRoutes({ router, mlLicense, routeGuard }: Rout
   /**
    * @apiGroup DataFrameAnalytics
    *
-   * @api {post} /api/ml/data_frame/analytics/job_exists Check whether jobs exists in current or any space
+   * @api {post} /api/ml/data_frame/analytics/jobs_exist Check whether jobs exists in current or any space
```

Comment:
I think it's `whether jobs exist` without the trailing `s`.

Reply:
Changed in https://github.com/elastic/kibana/pull/126914/commits/d2928234c6c593bbf0806dbf0e714fd2de2f05bd